### ISSUE TITLE
Get All example doesn't work

### ISSUE THREAD

**Issue by marcelloh** [nutsdb/nutsdb#545]
**Describe the bug**
In the documentation there is this part:
```
Get all
To scan all keys and values of the bucket stored, we can use GetAll function. For example:

if err := db.View(
    func(tx *nutsdb.Tx) error {
        bucket := "user_list"
        entries, err := tx.GetAll(bucket)
        if err != nil {
            return err
        }

        for _, entry := range entries {
            fmt.Println(string(entry.Key),string(entry.Value))
        }

        return nil
    }); err != nil {
    log.Println(err)
}
```
But entry.Key and entry.Value don't work anymore because entry is just []byte

**To Reproduce**
See the above (tested with 1.0.1)

**Expected behavior**
get the key of the record?

**What actually happens**
It doesn't compile

**please complete the following information :**
 - OS: [Mac OS 14..1.2]
 - NutsDB Version [1.0.1]

**Comment by bigboss2063** [comment]
Hi, because the new version does not release, the `Tx.GetMaxKey` and `Tx.GetMinKey` can't use for now. I'll release a new version that include this two API now. @marcelloh

**Comment by bigboss2063** [comment]
Hi, because the new version does not release, the `Tx.GetMaxKey` and `Tx.GetMinKey` can't use for now. I'll release a new version that include this two API now. @marcelloh

**Comment by bigboss2063** [comment]
I released v1.0.2.  There are many new APIs valid now.
